src/components/BottomSheet.tsx

```diff
@@ -36,14 +29,12 @@ const BottomSheet = ({ forwardedRef, title, description, children, testId }: Pro
     <GorhomBottomSheet
       ref={forwardedRef}
       index={-1}
-      snapPoints={animatedSnapPoints}
-      handleHeight={animatedHandleHeight}
-      contentHeight={animatedContentHeight}
+      enableDynamicSizing
```

Comment:
Do we want do add `snapPoints: ['CONTENT_HEIGHT'],` here in consistency with [Navigator.tsx](https://github.com/valora-inc/wallet/pull/4241/files#diff-37c5052bf8f3b1e84a2a8b2c37c9da7689b68089261c702b8031dc7f1db77b83R736)?

Reply:
no, the default snap point is implemented in the `@th3rdwave/react-navigation-bottom-sheet` library only. under the hood, `@th3rdwave/react-navigation-bottom-sheet` uses `@gorhom/bottom-sheet` but there are no default snap points required in `@gorhom/bottom-sheet`. the BottomSheet component uses the `@gorhom/bottom-sheet` lib directly.